### PR TITLE
Adjust the alert durations according to Kubevirt code

### PR DESCRIPTION
**Pull request by yuhaohaoyu**
Fixed following alert durations:

- VirtOperatorRESErrorsBurst : 5 minutes
- VirtOperatorRESErrorsHigh : 1 hour
- LowVirtOperatorCount : 1 hour

### CODE REVIEW

**Comment by yuhaohaoyu** [comment]
@sradco @erkanerol Please take a look. My teammate, Denys Schedrivyi, identified these mismatch and I just fixed them.

File: docs/runbooks/VirtOperatorRESTErrorsHigh.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Meaning
 
-More than 5% of the rest calls in virt-operator pods failed in the last hour. 
+More than 5% of the rest calls in virt-operator pods failed in the last 5 minutes.
```
Comment (dshchedr) [inline docs/runbooks/VirtOperatorRESTErrorsHigh.md:5]: this alert a bit confusing..
https://github.com/kubevirt/kubevirt/blob/9be692ed9807f9b1481dd4bbc0435c6d2cc79755/pkg/virt-operator/resource/generate/components/prometheus.go#L299-L310

we are watching here on 1hr metrics: _vec_by_virt_operators_failed_client_rest_requests_in_last_hour_ 
so alert will happen when we have **5% of failed packed in the last hour** for period more than 5 minutes in a row.

Comment (yuhaohaoyu) [reply docs/runbooks/VirtOperatorRESTErrorsHigh.md:5]: @dshchedr Thanks a million. How's this sentence sound? 

In the past hour, there are 5% of the rest calls in virt-operator pods failed for 5 minutes or longer.

Comment (dshchedr) [reply docs/runbooks/VirtOperatorRESTErrorsHigh.md:5]: hm, In my opinion we can avoid adding "5 min" here. I would go with the initial sentence:

More than 5% of the rest calls in virt-operator pods failed in the last hour

Comment (yuhaohaoyu) [reply docs/runbooks/VirtOperatorRESTErrorsHigh.md:5]: Asked a few people, this sentence seemed understandable:

In the past 5 minutes, the Kubevirt monitoring subsystem consistently observed 5% of failed rest calls in virt-operator pods for an hour.

**Pull request merged**